tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -143,6 +143,25 @@ def test__get_collections_with_auth__OK(self):
         res_auth = self.app.get("/curation/v1/collections", headers=self.make_owner_header())
         self.assertEqual(200, res_auth.status_code)
         self.assertEqual(6, len(res_auth.json["collections"]))
+        with self.subTest("The 'revising_in' attribute is \"NOT AUTHORIZED\" for unauthorized public collections"):
+            for c in res_auth.json["collections"]:
+                if c["id"] in (
+                    "test_collection_id_public_for_revision_one",
+                    "test_collection_id_public_for_revision_two",
+                ):
+                    self.assertEqual("NOT AUTHORIZED", c["revising_in"])
```

Comment:
couldn't we just leave this field out if they are not authorized? Do they need to know that there is a revision?

Reply:
It's set to `NOT_AUTHORIZED` for all public collections for which you are not allowed to view, regardless of whether there actually is an existing revision.  cc @brianraymor for comment

Reply:
If the design or requirements are unclear, I'd prefer to discuss in the issue and not the PR to capture outcomes. I'm going to respond in https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/2810.